source/_patterns/20-molecules/image-figure/image-figure.scss

```diff
@@ -103,3 +103,17 @@ Behaviour:
     margin-right: 16px;
   }
 }
+
+.image-figure--fullwidth-element.image-figure--fullwidth-element {
+  margin: 0;
+
+  .image {
+    background-color: $color-bg-media;
+  }
+
+  .image-figure__image,
+  .media-caption {
```

Comment:
.image und .media-caption gehören eigentlich nicht in dieses file - eigene modifier?